### DISCUSSION TITLE
QUDT license

### DISCUSSION THREAD

**Discussion by Mttbnchtt** [qudt/qudt-public-repo#482] [Q&A] [answered]
The [QUDT website](http://www.qudt.org/) says that the license is CC 4.0. 
![image](https://user-images.githubusercontent.com/35274169/160417342-907dbd98-2178-4a46-9a0d-691473943105.png)

However, the VOCAB_QUDT-SCALES-v2.0.ttl files, at row 26, says that the license for that file is CC Share Alice 3.0:
```
dc:rights "The QUDT Scale Vocabularies are issued under a Creative Commons Attribution Share Alike 3.0 United States License. Attribution should be made to NASA Ames Research Center and TopQuadrant, Inc."^^xsd:string ;
 
```

What does this mean? Does the general license govern all parts of the ontology? Or does the license (if stated) of a specific file takes precedence? Many thanks.

**Comment by steveraysteveray** [comment, marked as answer]
That is an oversight on our part. The intention is that all of the repository is licensed under CC-BY 4.0 International. We have scrubbed the contents of what we include in our official [release](https://github.com/qudt/qudt-public-repo/releases), but sections we haven't yet focused on (such as scales) have slipped through. I will try to clean that up when I get a chance.
